API/Repositories/Implementations/TournamentsRepository.cs

```diff
@@ -71,18 +71,19 @@ await QueryForParticipation(playerId, mode, dateMin, dateMax)
     }
 
     public async Task<IEnumerable<PlayerTournamentMatchCostDTO>> GetPerformancesAsync(int playerId,
-        int mode,
+        Ruleset ruleset,
         DateTime dateMin,
         DateTime dateMax,
-        int count,
-        bool bestPerformances)
+        int limit = 5,
+        bool bestPerformances = false)
```

Comment:
I'd consider using an enum here so we can support multiple cases in the future. Something like this:

```cs
public enum PerformanceResult
{
	Best,
	Worst,
	Recent
}
```